common/src/main/java/org/opensearch/ml/common/memorycontainer/MemoryConfiguration.java

```diff
@@ -87,7 +86,7 @@ public MemoryConfiguration(
         validateInputs(embeddingModelType, embeddingModelId, dimension, maxInferSize);
 
         // Assign values after validation
-        this.indexPrefix = indexPrefix == null ? "memory-" + UUID.randomUUID() : indexPrefix;
+        this.indexPrefix = indexPrefix == null ? ".plugins-ml" : indexPrefix;
```

Comment:
How about use specific prefix 
```suggestion
        this.indexPrefix = indexPrefix == null ? ".plugins-ml-memory" : indexPrefix;
```